### PR TITLE
[Feature] Support SmoothNet 

### PR DESCRIPTION
**Pull request by ailingzengzzz**
## Motivation

Support SmoothNet for pose smoothing

## Modification

- Add `SmoothNet` and `SmoothNetFilter`
- Support filter shared by multiple targets
- Update docs

## BC-breaking (Optional)

-

## Use cases (Optional)

-

## Checklist

**Before PR**:

- [ ] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) to create this PR.
- [ ] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) are used to fix the potential lint issues.
- [ ] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] New functionalities are covered by complete unit tests. If not, please add more unit tests to ensure correctness.
- [ ] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,6 +137,7 @@ A summary can be found in the [Model Zoo](https://mmpose.readthedocs.io/en/lates
 * [x] [UDP](https://mmpose.readthedocs.io/en/latest/papers/techniques.html#udp-cvpr-2020) (CVPR'2020)
 * [x] [Albumentations](https://mmpose.readthedocs.io/en/latest/papers/techniques.html#albumentations-information-2020) (Information'2020)
 * [x] [SoftWingloss](https://mmpose.readthedocs.io/en/latest/papers/techniques.html#softwingloss-tip-2021) (TIP'2021)
+* [x] [SmoothNet](/configs/_base_/filters/smoothnet_h36m.md) (arXiv'2021)
```
Comment (jin-s13) [inline README.md:140]: The link looks different from others. Shall we use something that starts with `https://`？

Comment (ly015) [reply README.md:140]: We don't have a readthedocs page or link for SmoothNet.

This PR only supports using SmoothNet as a temporal filter. The model training and testing are not included, thus there's no corresponding config files. And SmoothNet doesn't belong to any tasks we currently have, so the .md file is placed in `configs/_base_` and can't be collected when building the doc pages.

So I think we can link to the .md file here for now. And

Comment (ly015) [reply README.md:140]: We can add the readthedocs page for SmoothNet when the training is supported. Or we can reorganize the model zoo page to add something like 'plugin' as a new category.

**Review by ly015 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jin-s13) [reply README.md:140]: understood.

**Review by jin-s13 — APPROVED** [review]

**Pull request merged**